soc/ite/ec/it8xxx2/Kconfig.soc

```diff
@@ -50,3 +54,4 @@ config SOC
 	default "it82002aw" if SOC_IT82002_AW
 	default "it82202ax" if SOC_IT82202_AX
 	default "it82302ax" if SOC_IT82302_AX
+	default "it82202bw" if SOC_IT82202_BW
```

Comment:
I did not notice before, why is there an underscore in these Kconfig symbols? The names here should match e.g. `default "it82202_bw" if SOC_IT82202_BW` or `default "it82202bw" if SOC_IT82202BW`

Reply:
Remove underscore.